TFFoundationTests/Base/SequenceTests.swift

```diff
@@ -0,0 +1,83 @@
+//
+// MIT License
+//
+// Copyright (c) 2018 Trifork Kraków Office
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+//
+
+import XCTest
+@testable import TFFoundation
+
+class SequenceTests: XCTestCase {
+    
+    let empty = [Int]()
+    let oneElement = [1]
+    let twoElements = [1, 2]
+    
+    func testCountWhere() {
+        XCTAssertEqual(0, empty.count(where: { $0 == 1 }))
+        
+        XCTAssertEqual(1, oneElement.count(where: { $0 == 1 }))
+        XCTAssertEqual(0, oneElement.count(where: { $0 == 2 }))
```

Comment:
Do we need to reference `oneElement` property here? I think it would be clearer to have
 `XCTAssertEqual([1].count(where: { $0 == 2 }), 0)`
This way I don't need to look outside of the test method to check what is stored in `oneElement` property.

Reply:
Do you want to replace `empty`, `oneElement` and `twoElements` with their definitions in all the asserts? 

I see your point, but I also like avoiding repetitions. Maybe naming is not good enough. What about `none`, `one`, `oneTwo` ?